json-guides/circuit-breaker.json

```diff
@@ -11,63 +11,63 @@
-          "<h2>Circuit Breaker</h2>",
-          "The circuit breaker design pattern allows your microservice to fail immediately to prevent repeated calls that are likely to fail.<br>",
-          "The software circuit breaker operates much like an electrical circuit breaker. A <b><i>closed</i></b> circuit represents a fully functional system, and an <b><i>open</i></b> circuit represents an incomplete system. If a failure occurs, the circuit breaker triggers to open the circuit, removing the point of failure from the system.",
-          "<div class=\"backgroundConcepts\"><img src=\"/guides/iguide-circuit-breaker/html/interactive-guides/circuit-breaker/images/halfopen.svg\"><img src=\"/guides/iguide-circuit-breaker/html/interactive-guides/circuit-breaker/images/open.svg\"><img src=\"/guides/iguide-circuit-breaker/html/interactive-guides/circuit-breaker/images/closed.svg\"></div>",
-          "In a software circuit breaker, an additional <b><i>half-open</i></b> state exists. After the circuit is opened, it periodically changes to the half-open state, where it checks whether the failed component is restored and closes the circuit after it is considered safe and functional.<br>",
-          "<h2>Fallback</h2>",
-          "A fallback service runs when the main service fails. It can provide graceful failure or continued or partial operation of the original service.<br>",
-          "In our example, we use a fallback service with a CircuitBreaker policy. When the circuit breaker opens, subsequent requests for the service are immediately routed to the fallback service until the main service is restored."
-          ],
-          "displayType": "none"
-        },
-        {
-            "name": "Scenario",
-            "title": "Example: Bank scenario",
-            "description": [
-                "Imagine that your online banking application has many different microservices:",
-                "<ul><li>A service for checking your balance<li>A service for transferring money<li>A service for depositing money<li>A balance snapshot service that is used as a backup of data</ul>In the next step, you will try to check your balance on the bank's website."
-            ],
-            "content": [
+            "sections" : [
               {
-                "displayType":"pod",
-                "content": "<div class='bankScenarioImg'><img src='/guides/iguide-circuit-breaker/html/interactive-guides/circuit-breaker/images/intro.svg' alt='Online Banking Microservices'></div>"
+                "name": "backgroundConcepts",
+                "title": "Background Concepts",
+                "description": [
+                  "<b><i>Microservice</i></b> is an architecture that structures an application loosely from other services so it is independent and modular.<br>",
+                  "<h4><b><u>Circuit Breaker</u></b></h4>",
```

Comment:
what happened to all the images for the circuit breaker?

Reply:
I think something got overridden, i will check today.